generated/nidaqmx/_task_modules/in_stream.py

```diff
@@ -1087,7 +1087,7 @@ def read(self, number_of_samples_per_channel=READ_ALL_AVAILABLE):
             return numpy_array[:number_of_channels * samples_read]
         return numpy_array
 
-    def readall(self):
+    def read_all(self):
```

Comment:
This breaks compatibility. Shouldn't you deprecate the old name first?

That would involve adding a stub like this:
```python
@deprecation.deprecated(deprecated_in="0.10.0", details="Use read_all instead.")
def readall(self):
    return self.read_all()
```

Related: what is the plan for removing deprecated methods and properties? We haven't specified a version in which they will be removed.

Cc: @zhindes

Reply:
I like [PEP 387](https://peps.python.org/pep-0387/#backwards-compatibility-rules) rules:

> Wait for the warning to appear in at least two minor Python versions of the same major version, or one minor version in an older major version (e.g. for a warning in Python 3.10.0, you either wait until at least Python 3.12 or Python 4.0 to make the change).

For this, we would be adding the deprecation in 1.0, so we could remove in 1.2. For existing deprecations, we could remove them now (in 1.0). However, I think for simplicity we bucket all existing deprecations together and let them expire in 1.2.

Reply:
@DeborahOoi96 - do you mind creating an issue in this project to remove our current batch of deprecations in 1.2? (Unless Brad has a better idea!)

Reply:
Once we decide when the deprecations will be removed, we should also add `removed_in="1.2"` to them: https://pypi.org/project/deprecation/

This will update the deprecation messages in the documentation: https://nidaqmx-python.readthedocs.io/en/latest/ai_channel.html#nidaqmx._task_modules.channels.ai_channel.AIChannel.ai_acceld_b_ref

Reply:
Got it, I've made the necessary changes and added test cases for the deprecated old function name as well.